guide/interactions/buttons.md

```diff
@@ -81,6 +81,15 @@ client.on('interactionCreate', async interaction => {
 });
 ```
 
+::: warning
+If you're using typescript you'll need to specify the type of components your action row holds. This can be done by specifying the generic parameter in `ActionRowBuilder`.
```

Comment:
```suggestion
If you're using typescript you'll need to specify the type of components your action row holds. This can be done by specifying the component builder you will add to it using a generic parameter in `ActionRowBuilder`.
```